connectivity/FEATURE_BLE/source/cordio/CMakeLists.txt

```
@@ -1,14 +1,14 @@
 # Copyright (c) 2020 ARM Limited. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
-target_include_directories(mbed-os
+target_include_directories(mbed-os-ble
```

```suggestion
target_include_directories(mbed-os-ble-cordio
```
See the directory this file is. All sources and include dirs should be added to the `mbed-os-ble-cordio`.